### PR TITLE
FIX tests under windows

### PR DESCRIPTION
**Pull request by ogrisel**
This should make the tests pass under Windows (issue reported in #230).

We still need to setup proper Windows CI.

### CODE REVIEW

File: tests/cloudpickle_test.py [hunk 1]
```diff
@@ -487,7 +491,7 @@ def func():
         '''
```
Comment (HyukjinKwon) [inline tests/cloudpickle_test.py]: Out of curiosity, wouldn't it work if we use raw string here, like `child_process_script = r''' ... '''`?

Comment (HyukjinKwon) [reply tests/cloudpickle_test.py]: Oops, nvm, the problem is the path.

**Comment by ogrisel** [comment]
Thanks for the review @HyukjinKwon. I merged to move on. I will see if I can configure travis for windows in a new PR.

**Comment by HyukjinKwon** [comment]
@ogrisel, FYI, there's CI called AppVeyor for Windows. I took a look a lone ago before to make it work with Travis but ended up with AvppVeyor (few years ago). So. currently, Apache Spark uses AppVeyor for Windows build.

**Comment by ogrisel** [comment]
Yes I know AppVeyor but since travis now supports windows, I would rather maintain just a single configuration file for all platforms.

**Comment by HyukjinKwon** [comment]
Ohh I see thanks!

**Pull request merged**